neural_compressor/torch/quantization/autotune.py

```diff
@@ -33,7 +29,7 @@
 
 def get_default_tune_config() -> TuningConfig:
     # TODO use the registered default tuning config in the next PR
-    return TuningConfig(quant_configs=[GPTQConfig(weight_bits=[4, 8]), RTNConfig(weight_bits=[4, 8])])
+    return TuningConfig(quant_configs=[GPTQConfig(weight_bits=[4, 8]), RTNConfig(bits=[4, 8])])
```

Comment:
GPTQConfig(bits=[4, 8])

Reply:
GPTQ refactor is not contained in this PR